ticketauth/build.gradle.kts

```diff
@@ -9,10 +11,15 @@ android {
 
     defaultConfig {
         minSdk = 26
-        targetSdk = 31
+        targetSdk = 32
 
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles("consumer-rules.pro")
+        buildConfigField(
+            "String",
+            "TICKETAUTH_VERSION",
+            "\"1.1.0\""
```

Comment:
We might need to add a release section in the `README` that specifies one need to update this before a release.

Reply:
good input, will do :+1: